packages/core/data-transfer/src/utils/index.ts

```diff
@@ -2,3 +2,4 @@ export * as encryption from './encryption';
 export * as stream from './stream';
 export * as json from './json';
 export * as schema from './schema';
+export * as transaction from './transaction';
```

Comment:
I would argue that this transaction util is more linked to Strapi than common utils. But this is minor.